tests/unit/Extension/Footnote/Renderer/FootnoteBackrefRendererTest.php

```diff
@@ -64,4 +64,18 @@ public function testClassConfiguration(): void
 
         $this->assertStringContainsString('class="my-custom-class"', $output);
     }
+
+    public function testTextConfiguration(): void
+    {
+        $renderer = new FootnoteBackrefRenderer();
+        $renderer->setConfiguration(new Configuration(['footnote' => ['backref_symbol' => 'Go back.']]));
+
+        $fakeReference   = new Reference('label', 'dest', 'title');
+        $footnoteBackref = new FootnoteBackref($fakeReference);
+
+        $output = $renderer->render($footnoteBackref, new FakeChildNodeRenderer());
+
+        $this->assertStringContainsString('Go back.', $output);
```

Comment:
Minor nit/personal preference: generally, I try to avoid assertions like this as it doesn't guarantee the expected position of where a new configurable element should be injected. In other tests, this kind of assertion may make more sense, but I think this one should assert the entire HTML output to ensure the symbol isn't accidentally and arbitrarily moved.

Reply:
P.S. It should also likely test for special HTML characters to ensure they're escaped properly.

Reply:
@markcarver thanks, I'll add a test for special HTML characters. Can you show me an example of what you mean for the first point? I didn't see anything like that for the Heading Permalink extension, and there are a bunch of snapshot and spec tests that do assert the entire output, so here I was mostly just testing that the config option works.

Reply:
@bakerkretzmar, sure: https://github.com/thephpleague/commonmark/blob/3b22eb3be0096a20f6edd8ab6a1516d437a9839e/tests/functional/Extension/HeadingPermalink/HeadingPermalinkExtensionTest.php#L45-L74